posts/ja/2022-11-01-letstry-instant-on-intel-mac-en.adoc

```diff
@@ -0,0 +1,280 @@
+---
+layout: post
+title: Let's try Open Liberty InstantOn on Intel Mac
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/takakiyo
+author_github: https://github.com/takakiyo
+seo-title: Let's try Open Liberty InstantOn on Intel Mac - OpenLiberty.io
+seo-description: Let's try Open Liberty InstantOn on Intel Mac
+blog_description: "Let's try Open Liberty InstantOn on Intel Mac"
+//open-graph-image: https://openliberty.io/img/blog/instantonperf.png
+//blog-available-in-languages: ['ja']
+additional_authors: 
+- name: Hiroko Takamiya
+  github: https://github.com/una-tapa
+  image: https://avatars0.githubusercontent.com/una-tapa
+---
+= Let's try Open Liberty InstantOn on Intel Mac
+Takakiyo Tanaka <https://github.com/takakiyo>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+== What is Liberty InstantOn ? 
+
+Liberty InstantOn was announced on September 29, 2022. It is a technology that launches a full-featured Java EE application server in hundreds of milliseconds.
+
+When using Serverless technology in a container environment, the startup time is important. The response time includes the time it takes for the container process to start.  So the startup time directly affects the quality of service.
+
+Many efforts have been made to reduce the startup time for Java applications. For example, one of the efforts is to use the capabilities of GraalVM to transform the application into a native executable. However, GraalVM does not allow all Java features to be used, so it is not possible to convert widely used Java EE applications into executables.
+
+Liberty InstantOn takes a totally different approach to reduce the startup time. It takes advantage of the Linux CRIU (Checkpoint/Restore In Userspace) technology. CRIU is a set of functions for recording (Checkpoint) the information of the running process in a file and restoring (Restore) the process from the file. 
+
+Open Liberty/WebSphere Liberty with an OpenJ9 VM incorporates CRUI technology to get a Checkpoint when the application finishes initializing. By committing the container in this state and resuming it from there, it can be started in a lightning fast time.
+
+== Let's try InstantOn on Intel Mac
+
+The InstantOn steps in the blog article link:https://openliberty.io/blog/2022/09/29/instant-on-beta.html[Liberty InstantOn startup for cloud native Java applications] are  based on the Red Hat Enterprise Linux. The same experience can be achieved on an Intel Mac with macOS Monterey (version 12.x). This article shares the steps to experience the exciting technology on Mac. 
```

Comment:
This begs the question.  Can this work with M1 Rosetta? If we know it doesn't work then we should mention it.

Reply:
Currently trying to chat with the techsales team. leaving the conversation open.

Reply:
Adding "Currently, InstantOn feature is not available on M1 aarch64 architecture. "